.golangci.yml

```diff
@@ -81,3 +82,7 @@ issues:
         # For some reason, this linter is coming up with several
         # false positives in this file ¯\_(ツ)_/¯
         - bodyclose
+    # Tests check for global state via goroutine count
+    - path: client_test.go
+      linters:
+        - tparallel
```

Comment:
Let's merge this with the above exception.

```diff
     - path: client_test\.go
       linters:
         # These tests examine goroutines to make sure all resources
         # are cleaned up, which doesn't work when run in parallel.
         - paralleltest
+        - tparallel
         # For some reason, this linter is coming up with several
         # false positives in this file ¯\_(ツ)_/¯
         - bodyclose
```

Reply:
This seems to not be an issue anymore, resolved.